src/shapes/Group.ts

```diff
@@ -300,13 +308,14 @@ export class Group
     selected: T,
     { target: object }: ObjectEvents[T extends true ? 'selected' : 'deselected']
   ) {
+    const activeObjects = this._activeObjects;
     if (selected) {
-      this._activeObjects.push(object);
+      activeObjects.push(object);
       this._set('dirty', true);
-    } else if (this._activeObjects.length > 0) {
-      const index = this._activeObjects.indexOf(object);
+    } else if (activeObjects.length > 0) {
+      const index = activeObjects.indexOf(object);
       if (index > -1) {
-        this._activeObjects.splice(index, 1);
+        activeObjects.splice(index, 1);
```

Comment:
I am not sure this even used

Reply:
i just wanted to save some bytes with minification. I think is used for rendering and moving active selections